examples/module/controller/SE3_controller.py

```diff
@@ -0,0 +1,60 @@
+import torch
+from pypose.module.controller import Controller
+from examples.module.controller_parameters_tuner.commons \
+  import hat, vee, quaternion_2_rotation_matrix
+
+
+class SE3Controller(Controller):
```

Comment:
Suggest using geometric control naming, and move to the module rather than staying in example